<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Tasks">
    <property name="title">
     <string>Tas&amp;ks</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOpenTaskFolder"/>
    <addaction name="actionTaskManager"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenTaskFile"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Tasks"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDefaultTaskFolder">
   <property name="text">
    <string>Open &amp;Default Task Folder</string>
   </property>
  </action>
  <action name="actionOpenTaskFolder">
   <property name="text">
    <string>&amp;Open Task Folder</string>
   </property>
  </action>
  <action name="actionTaskManager">
   <property name="text">
    <string>&amp;Task Manager</string>
   </property>
  </action>
  <action name="actionOpenTaskFile">
   <property name="text">
    <string>&amp;Open Task File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
